Library/Homebrew/utils/analytics.sh

```diff
@@ -14,17 +22,16 @@ setup-analytics() {
   if [[ "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsmessage)" != "true" ||
         "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsdisabled)" = "true" ]]
   then
-    [[ -f "$HOMEBREW_ANALYTICS_USER_UUID_FILE" ]] && rm -f "$HOMEBREW_ANALYTICS_USER_UUID_FILE"
+    git config --file="$HOMEBREW_REPOSITORY/.git/config" --unset-all homebrew.analyticsuuid
```

Comment:
Might want to pull `$HOMEBREW_REPOSITORY/.git/config` into a shared variable in `setup`?